### PR TITLE
Feature/user api crud

### PR DESCRIPTION
**Pull request by AhtishamShahid**

### CODE REVIEW

File: .idea/dataSources.local.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (waheedahmed) [inline .idea/dataSources.local.xml]: Add `.idea/*` to gitignore.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  - docker-compose build
+  - docker-compose up -d
+#before_script: # => 6
+#  - mysql -e 'create database test;' -u root
+script: # => 7
+  - sleep 40
+  - docker exec -t recipeapp_web_1 python manage.py test
+  - docker exec -t recipeapp_web_1 pylint --load-plugins pylint_django recipe/
+  - docker exec -t recipeapp_web_1 pylint --load-plugins pylint_django users/
+  - docker exec -t recipeapp_web_1 pylint --load-plugins pylint_django root/
```
Comment (waheedahmed) [inline .travis.yml]: Always add a new line at the end of file.

Comment (AhtishamShahid) [reply .travis.yml]: Fixed

File: .idea/dataSources.local.xml (hunk 1 above)
Comment (AhtishamShahid) [reply .idea/dataSources.local.xml]: Fixed

**Pull request closed without merging**